kroxylicious/src/main/templates/BodyDecoder.ftl

```diff
@@ -0,0 +1,80 @@
+<#--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+/*
```

Comment:
What's the reason for adding the full ASF license terms to the generated source code?  Why not our shorter Kroxylicious license?  I know you've followed existing conventions, but I am questioning the practice.

Reply:
It looks like it's been there from the beginning, I can see [a commit](https://github.com/kroxylicious/kroxylicious/commit/67e712e32e33e28a18c7917f05fb05ea4a28008f#diff-ff474243c23ad02e08be355e41c489c8e2a88c17838cf9fdeaba368ec2142ec0) early on that changed from the full license to the smaller version for other classes, but maybe there's some licensing difference since it's generated?

@gunnarmorling do you know if it needs to be there?